challenges/multi-bracket-validation/README.md

```diff
@@ -0,0 +1,30 @@
+# Multi Bracket Validation
+**Author** : George Ceja
+**Version**: 0.1.0
+
+
+## Overview
+create a function called multi_bracket_validation that validates the input and
+returns true or false if brackets have even pairs.
+
+Types of Brackets to check for ---> (), [], {}
+
+Structure and Testing
+Utilized the Single-responsibility principle: Methods created are clean,
+reusable, and abstract components.
+
+
+## Getting Started
+Create a branch in your repository called multi-bracket-validation
+Create file called multi_bracket_validation.py
```

Comment:
This section is supposed to be how the _user_ should get started using your code, not how you got started.